sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clientTelemetry/AzureVMMetadata.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.clientTelemetry;
+
+public class AzureVMMetadata {
```

Comment:
VM SKU woudl be very helpful as well?

Reply:
Discussed offline , added hostEnvInfo in format "Linux|18.04-LTS|Standard_F4s|AzurePublicCloud"